rust/cadeau/src/xmf/vpx/mod.rs

```diff
@@ -98,7 +111,8 @@ impl VpxPacket<'_> {
         if frame_ptr.is_null() {
             None
         } else {
-            Some(VpxFrame { ptr: frame_ptr })
+            // SAFETY: The frame pointer is not null and is valid, since the packet is valid.
```

Comment:
question: Can you explain how the packet relates to the pointer of `VpxImage`?

Reply:
No, the frame is a deep copy when the packet is a frame packet. So it is owned by the VpxFrame struct. I'll add this to comment.